app/src/main/java/com/asfoundation/wallet/service/AccountWalletService.kt

```diff
@@ -50,6 +78,26 @@ class AccountWalletService(private val walletInteract: FindDefaultWalletInteract
     return signature.toHex()
   }
 
+  fun find(): Single<Wallet> {
+    return walletRepository.defaultWallet
+        .onErrorResumeNext {
+          walletRepository.fetchWallets()
+              .filter { wallets: Array<Wallet?> -> wallets.isNotEmpty() }
+              .map { wallets: Array<Wallet> ->
+                wallets[0]
+              }
+              .flatMapCompletable { wallet: Wallet ->
+                walletRepository.setDefaultWallet(wallet.address)
+              }
+              .andThen(
+                  walletRepository.defaultWallet)
+        }
+  }
+
+  fun create(): Single<Wallet> {
+    return walletCreatorInteract.create()
+  }
```

Comment:
we could use a single line expression =>   fun create(): Single<Wallet> = walletCreatorInteract.create()